ckan/ckanext/ckanext-registrydata/ckanext/registrydata/schemas/dataset.json

```diff
@@ -4,31 +4,327 @@
   "about_url": "http://github.com/vrk-kpa/registrydata",
   "dataset_fields": [
     {
-      "field_name": "title",
+      "field_name": "title_translated",
       "label": "Title",
-      "preset": "title",
-      "form_placeholder": "eg. My dataset"
+      "preset": "fluent_core_title_translated",
+      "form_languages": ["fi", "en", "sv"],
+      "form_placeholder": "e.g. Finnish names",
+      "required_languages": ["fi", "sv"],
```

Comment:
Should required languages be done with default lang and additional languages instead ?

Reply:
I did consider it, but I think it would be more confusing that way. The language options are enumerated above. It makes sense to configure the model based on those. If it was just the default language it would make sense.